source-academy/cadet-frontend
# update Inspector's list of built ins

### Description

This list keeps track of names of all the built ins so that they will not be displayed when inspecting.

### Type of change


### How to Test

Run with breakpoint. Notice on the live version you get some built ins such as `yellow` displayed under global scope. This version does not display them.

### Checklist
